sdk/eventhub/azure-eventhub/samples/async_samples/sample_code_eventhub_async.py

```diff
@@ -18,12 +18,16 @@
 
 import logging
 import asyncio
+import os
+from typing import TYPE_CHECKING, Optional, List
+from azure.eventhub.aio import EventHubConsumerClient, EventHubProducerClient, EventHubSharedKeyCredential
 
+if TYPE_CHECKING:
+    from azure.eventhub import EventData, EventDataBatch, PartitionContext
 
-def example_create_async_eventhub_producer_client():
+
+def example_create_async_eventhub_producer_client() -> EventHubProducerClient:
     # [START create_eventhub_producer_client_from_conn_str_async]
-    import os
-    from azure.eventhub.aio import EventHubProducerClient
```

Comment:
We left these here so that they'd be included in the documentation samples, which are indicated by the `# [START ...]` and `# [END ...]` comments. I think we'd want the imports here rather than up top.